--- a/modules/holoviz/src/glfw_window.cpp	2026-07-02 19:37:17.000000000 -0400
+++ b/modules/holoviz/src/glfw_window.cpp	2026-09-24 19:17:29.989395251 -0400
@@ -677,7 +677,7 @@
   VkSurfaceKHR surface;
   const vk::Result result =
       vk::Result(glfwCreateWindowSurface(instance, impl_->window_, nullptr, &surface));
-  vk::resultCheck(result, "Failed to create glfw window surface");
+  vk::detail::resultCheck(result, "Failed to create glfw window surface");
   return surface;
 }
 
--- a/modules/holoviz/src/vulkan/vulkan_app.cpp	2026-07-02 19:37:17.000000000 -0400
+++ b/modules/holoviz/src/vulkan/vulkan_app.cpp	2026-09-24 19:17:29.989655104 -0400
@@ -942,7 +942,7 @@
   // submit staged transfers
   const vk::Result result =
       vk::Result(nvvk_.transfer_batch_submission_.execute(transfer_job.fence_.get(), 0b0000'0001));
-  vk::resultCheck(result, "Failed to execute batch submission");
+  vk::detail::resultCheck(result, "Failed to execute batch submission");
 
   // next graphics submission must wait for transfer completion
   nvvk_.batch_submission_.enqueueWait(transfer_job.semaphore_.get(),
@@ -1019,7 +1019,7 @@
 
         it->fence_triggered_ = true;
       } else if (result != vk::Result::eNotReady) {
-        vk::resultCheck(result, "Failed to get upload fence status");
+        vk::detail::resultCheck(result, "Failed to get upload fence status");
       }
     }
 
@@ -1037,7 +1037,7 @@
           it = next;
           continue;
         } else if (result != vk::Result::eNotReady) {
-          vk::resultCheck(result, "Failed to get frame fence status");
+          vk::detail::resultCheck(result, "Failed to get frame fence status");
         }
       } else {
         // this is a active transfer buffer, keep it; the frame fence will be set after the call to
@@ -1071,7 +1071,7 @@
   if (result != vk::Result::eSuccess) {
     // This allows Aftermath to do things and exit below
     usleep(1000);
-    vk::resultCheck(result, "Failed to wait for frame fences");
+    vk::detail::resultCheck(result, "Failed to wait for frame fences");
     exit(-1);
   }
 
@@ -1109,7 +1109,7 @@
 
   const vk::Result result =
       vk::Result(nvvk_.batch_submission_.execute(wait_fences_[image_index].get(), 0b0000'0001));
-  vk::resultCheck(result, "Failed to execute batch submission");
+  vk::detail::resultCheck(result, "Failed to execute batch submission");
 
   if (!(render_flags_ & RenderFlags::DONT_SWAP_BUFFERS)) {
     // Presenting frame
@@ -2488,7 +2488,7 @@
     if (result != vk::Result::eSuccess) {
       // This allows Aftermath to do things and exit below
       usleep(1000);
-      vk::resultCheck(result, "Failed to wait for frame fences");
+      vk::detail::resultCheck(result, "Failed to wait for frame fences");
       exit(-1);
     }
 
@@ -2545,7 +2545,7 @@
   // submit the command buffer
   const vk::Result result =
       vk::Result(nvvk_.batch_submission_.execute(read_job.fence_.get(), 0b0000'0001));
-  vk::resultCheck(result, "Failed to execute batch submission");
+  vk::detail::resultCheck(result, "Failed to execute batch submission");
 
   // copy the buffer to CUDA memory
   {
@@ -2633,7 +2633,7 @@
   if (result == vk::Result::eTimeout) {
     return false;
   }
-  vk::resultCheck(result, "Failed to wait for present");
+  vk::detail::resultCheck(result, "Failed to wait for present");
   return true;
 }
 
@@ -2676,7 +2676,7 @@
     if (result == vk::Result::eTimeout) {
       return false;
     }
-    vk::resultCheck(result, "Failed to wait for display event");
+    vk::detail::resultCheck(result, "Failed to wait for display event");
     return true;
   } catch (const vk::Error& e) {
     // Catch is intentionally broad (any vk-hpp error, not just vk::UnknownError):
